Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -554,6 +554,13 @@ void ProgramShaderCache::CreateHeader()
 	break;
 	}
 
+	const char* earlyz_string = g_ogl_config.bSupportsEarlyFragmentTests ?
+	            "#extension GL_ARB_shader_image_load_store : enable\n"
+	            "#define FORCE_EARLY_Z layout(early_fragment_tests) in\n" :
+	            // Fallback method:
```

Comment:
NB: At least some of these questions are answered in PixelShaderGen, but it's not clear how anyone would discover the comment in that file when spotting this comment.